kiota_abstractions/request_headers.py

```diff
@@ -0,0 +1,185 @@
+from __future__ import annotations
+
+from typing import Dict, List, Set, Union
+
+
+class RequestHeaders():
```

Comment:
Maybe call that class headers collection instead since it can be used for response headers?

Reply:
Makes sense. Updated.